drivers/dma/dma_mcux_edma.c

```diff
@@ -275,6 +274,27 @@ static void dma_mcux_edma_error_irq_handler(const struct device *dev)
 	barrier_dsync_fence_full();
 #endif
 }
+
+#if DT_ANY_INST_HAS_PROP_STATUS_OKAY(channels_shared_irq_mask)
+static void dma_mcux_edma_multi_channels_irq_handler(const struct device *dev, uint32_t idx,
+			uint32_t *buf, uint32_t step)
```

Comment:
I'd suggest renaming the `step` variable to `mask_width` or something similar- that might make this code block a bit easier to follow IMO.

Reply:
Update to use mask_width variable and modify the related macro name.